stl/inc/limits

```diff
@@ -1093,10 +1093,10 @@ _NODISCARD int _Checked_x86_x64_countr_zero(const _Ty _Val) noexcept {
 
 #if _HAS_POPCNT_INTRINSICS
 template <class _Ty>
-_NODISCARD int _Checked_x86_x64_popcount(const _Ty _Val) noexcept {
+_NODISCARD int _Checked_x86_x64_popcount(const _Ty _Val, const int _Isa_available) noexcept {

```

Comment:
We could pass this as a template parameter and use `if constexpr`

Reply:
No. `__isa_available` is runtime detected.